src/Json_decode.mli

```diff
@@ -13,7 +13,11 @@ third-party libraries.
 type 'a decoder = Js.Json.t -> 'a
 (** The type of a decoder combinator *)
 
-exception DecodeError of string
+type error = Json_error of string | Unpexpected_variant of string
```

Comment:
Doesn't seem that the second branch of the variant is used, but:

```suggestion
type error = Json_error of string | Unexpected_variant of string
```

Reply:
The second case is generated by ppx